packages/form-js-editor/package.json

```diff
@@ -43,6 +43,7 @@
   },
   "dependencies": {
     "@bpmn-io/form-js-viewer": "^0.10.0-alpha.0",
+    "@bpmn-io/properties-panel": "0.25.0",
```

Comment:
Do we need to pin it? We don't do that elsewhere if there is no need.

Reply:
We don't have to, it's a habit I have to guarantee more stable installations
But I can change it to keep consistent

Reply:
Let's do so until we see the need to pin it 👍